sources/us/ny/statewide.json

```diff
@@ -14,6 +14,14 @@
     "conform": {
         "type": "geojson",
         "street": "CompleteStreetName",
-        "number": "AddressNumber"
+        "number": ["PrefixAddressNumber", "AddressNumber", "SuffixAddressNumber"],
+        "unit": "SubAddress",
+        "city": "ZipName",
+        "region": "State",
+        "postcode": "ZipCode",
```

Comment:
We can't use the `ZipCode` field because it is apparently filled using proprietary Navteq data. See https://github.com/openaddresses/openaddresses/issues/1473.